homeassistant/components/google_assistant/trait.py

```diff
@@ -2054,7 +2057,10 @@ def _verify_pin_challenge(data, state, challenge):
     if not challenge:
         raise ChallengeNeeded(CHALLENGE_PIN_NEEDED)
 
-    if challenge.get("pin") != data.config.secure_devices_pin:
+    pins = data.config.secure_devices_pin
+    if not isinstance(pins, list):
+        pins = [pins]
+    if not challenge.get("pin") in pins:
```

Comment:
Use `not in`

Reply:
Fixed in https://github.com/home-assistant/core/pull/64429/commits/3859310c4ee598780fc55b0aff088c16ff264735